vscode-dotnet-runtime-extension/.vscodeignore

```diff
@@ -0,0 +1,8 @@
+.vscode/**
```

Comment:
All these wouldn't be awful to keep at top level would they? Otherwise a new extension would just replicate them all anyhow